pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -30,15 +30,16 @@ type srcDstDetails struct {
 	egressEnabled  bool
 	// the connection between src to dst, in case the connection was not part of the query;
 	// the part of the connection relevant to the query otherwise.
-	conn   *connection.Set
-	router RoutingResource // the router (fip or pgw) to external network; nil if none
+	conn           *connection.Set
+	externalRouter RoutingResource // the router (fip or pgw) to external network; nil if none or not relevant
+	tgwRouter      RoutingResource // the tgw router between src and dst from different VPCs; nil if none or not relevant
```

Comment:
1. Renamed to crossVpcRouter
2. Good question. Is it possible that connection between two VPCs is enabled by more than one tgw Router or by more than one tgw Connection in the same Router? @adisos @kyorav 
Currently the implementation assumes one of each (If indeed more than one router/connection is possible I would defer it to a different PR, perhaps even to V0.5 and for now just add a comment).

Reply:
I don't think you can connect two TGSs sequentially, but you can connect a VPC to two TGWs. So if two VPCs are both connected to the same two TGWs there are two possible routes connecting them. In this situation IBM does not guarantee the behavior, i.e. you don't know which route would be taken and this may change arbitrarily. I don't know if there is an actual use-case for this. It is probably OK to raise a warning, or even an error saying we don't support this scenario.

Reply:
We already have an error in this case - since it will result in the `<src, dst>` being found in more than one `multiVpc` config. So in this stage I'll just clarify the error message and add that this scenario is not supported (yet).

Reply:
done